scss/_patterns_image-grid.scss

```diff
@@ -0,0 +1,15 @@
+// Image grid to showcase group of images
+@mixin vf-p-p-image-grid {
+
+  .p-p-image-grid {
+    @extend .clearfix;
+    text-align: center;
+    display: block;
+
+    &__img {
```

Comment:
We need to control the max-width and height here. As rarely have the correct sized images. Don't SVGs are used and they can be large.